lib/cmds/space_cmds/generate_cmds/migration.js

```diff
@@ -0,0 +1,237 @@
+import { getContext } from '../../../context'
+import { assertLoggedIn, assertSpaceIdProvided, assertEnvironmentIdProvided } from '../../../utils/assertions'
+import { handleAsyncError as handle } from '../../../utils/async'
+import { log, success } from '../../../utils/log'
+
+const recast = require('recast')
```

Comment:
can you change this to `import recast from 'recast'` to keep the same style, also from the other dependencies